workers/branding/src/index.ts

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import { Router, error, cors } from 'itty-router'
+import { WorkerEntrypoint } from 'cloudflare:workers'
+import type { Branding, BrandingMap } from '@hcengineering/core'
+
+import type { Env } from './env'
+
+const { preflight, corsify } = cors({
+  maxAge: 86400
+})
+
+const router = Router<Request>({
+  before: [preflight],
+  catch: error,
+  finally: [(r: any) => r ?? error(404), corsify]
+})
+
+async function getBrandings(env: Env): Promise<BrandingMap> {
+  // Note: up to 100 keys can be listed at once. If more, use cursor.
+  const keys = (await env.KV.list()).keys
```

Comment:
what do you think about using prefix for all branding keys? so you can do `env.KV.list({ prefix: 'branding' })`

Reply:
not share, but just in case you have to put something else into this KV but related to brandings

Reply:
Thanks, updated